ani-cli

```diff
@@ -18,7 +18,7 @@
 # Project repository: https://github.com/pystardust/ani-cli
 
 # Version number
-VERSION="1.7.1"
+VERSION="1.7.2"
```

Comment:
Bump to 1.7.6 to account for missed bumps

Reply:
done